src/app/garfile/arrival/index.njk

```diff
@@ -57,7 +57,7 @@
                   <label class="govuk-label govuk-date-input__label" for="arrivalMonth">
                     <span class="govuk-visually-hidden">{{__('field_arrival_hint')}}</span>{{__('field_month')}}
                   </label>
-                  <span id="arrival-month-hint" class="govuk-visually-hidden">{{__('field_departure_month_hint')}}</span>
+                  <span id="arrival-month-hint" class="govuk-visually-hidden">{{__('field_month_hint')}}</span>
```

Comment:
This was actually missing! So screen reader was probably reading "field departure month hint" out. Whoops